test/weet.test.js

```diff
@@ -34,10 +40,21 @@ describe.only('Weets', () => {
       response = await request(app)
         .post('/weets')
         .set('Authorization', `Bearer ${regularToken}`);
+
       axios.get.mockImplementationOnce(() => Promise.resolve(createLongWeetInput));
       responseLong = await request(app)
         .post('/weets')
         .set('Authorization', `Bearer ${regularToken}`);
+
+      const getWeetsSpy = jest.spyOn(WeetService, 'getWeets');
+      getWeetsSpy.mockImplementation(getWeetsListMock);
+      responseWeetList = await request(app)
+        .get('/weets?page=0&limit=5')
+        .set('Authorization', `Bearer ${regularToken}`);
+
+      badResponseWeetList = await request(app)
+        .get('/weets?page=string&limit=5')
+        .set('Authorization', `Bearer ${regularToken}`);
```

Comment:
You can put all this in the describe for the get method tests. This is in order that the tests are more ordered.